frontend/src/components/feed/Feed.js

```diff
@@ -5,6 +5,10 @@ const Feed = ({ navigate }) => {
   const [posts, setPosts] = useState([]);
   const [token, setToken] = useState(window.localStorage.getItem("token"));
 
+  function orderByDate (posts) {
+    return posts.sort((a,b) => new Date(a.createdAt) - new Date(b.createdAt)).reverse()
```

Comment:
Wow... I really need to think about this!
What is createdAt? A string I guess?

Reply:
yes! it comes back from the database as a string but converts perfectly to a Date object in Javasript - it is formatted correctly already